Index: bundles/CoreBundle/src/DependencyInjection/Configuration.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bundles/CoreBundle/src/DependencyInjection/Configuration.php b/bundles/CoreBundle/src/DependencyInjection/Configuration.php
--- a/bundles/CoreBundle/src/DependencyInjection/Configuration.php
+++ b/bundles/CoreBundle/src/DependencyInjection/Configuration.php	(date 1694679108847)
@@ -1463,6 +1463,18 @@
                                             ->arrayNode('options')
                                                 ->children()
                                                     ->scalarNode('label')->info('Nice name for the Pimcore backend.')->end()
+                                                    // Studio1 start
+                                                    ->arrayNode('data')
+                                                        ->children()
+                                                            ->scalarNode('type')->info('')->defaultValue('')->end()
+                                                            ->scalarNode('class')->info('')->defaultValue('')->end()
+                                                            ->scalarNode('attribute')->info('')->defaultValue('')->end()
+                                                            ->scalarNode('languages')->info('')->defaultValue('')->end()
+                                                            ->scalarNode('languagesSetAll')->info('')->defaultValue(false)->end()
+                                                            ->scalarNode('value')->info('')->defaultValue('')->end()
+                                                        ->end()
+                                                    ->end()
+                                                    // Studio1 end
                                                     ->arrayNode('notes')
                                                         ->children()
                                                             ->booleanNode('commentEnabled')->defaultFalse()->info('If enabled a detail window will open when the user executes the transition. In this detail view the user be asked to enter a "comment". This comment then will be used as comment for the notes/events feature.')->end()
