Index: pimcore/vendor/pimcore/workflow-designer/src/Resources/public/js/pimcore/configuration/item/placeSettings.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/pimcore/vendor/pimcore/workflow-designer/src/Resources/public/js/pimcore/configuration/item/placeSettings.js b/pimcore/vendor/pimcore/workflow-designer/src/Resources/public/js/pimcore/configuration/item/placeSettings.js
--- a/pimcore/vendor/pimcore/workflow-designer/src/Resources/public/js/pimcore/configuration/item/placeSettings.js	
+++ b/pimcore/vendor/pimcore/workflow-designer/src/Resources/public/js/pimcore/configuration/item/placeSettings.js	(date 1680598907695)
@@ -4,7 +4,7 @@
 
 
     initialize: function(placeId, placeData, callback) {
-
+        placeData.id = placeId;
         const data = placeData || {};
         placeData.permissions = placeData.permissions || [];
 
@@ -64,6 +64,13 @@
             },
             items: [
                 {
+                    xtype: 'textfield',
+                    fieldLabel: t('id'),
+                    name: '__ignore_name',
+                    value: placeData.id,
+                    readOnly: true,
+                    width: 540,
+                }, {
                     xtype: 'textfield',
                     fieldLabel: t('label'),
                     name: 'label',
